documentation/blog/2023-12-01-whats-new-in-svelte-december-2023.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "What's new in Svelte: December 2023"
+description: 'Svelte 5.0 announced at Svelte Summit Fall 2023'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+Svelte Summit Fall 2023 was epic! Featuring talks from across the ecosystem and the launch of the beta version of Svelte 5.0! Individual videos for each talk can be found in [this playlist](https://www.youtube.com/watch?v=RT0aemVpvVE&list=PL8bMgX1kyZTh_wu-CruPkxk6sMbXXluJK), but the whole talk can be found [here](https://www.youtube.com/watch?v=pTgIx-ucMsY) (timestamps below):
```

Comment:
We debated elsewhere how exactly to frame this, and we settled on unstable preview
```suggestion
Svelte Summit Fall 2023 was epic! Featuring talks from across the ecosystem and the launch of the (unstable!) preview version of Svelte 5! Individual videos for each talk can be found in [this playlist](https://www.youtube.com/watch?v=RT0aemVpvVE&list=PL8bMgX1kyZTh_wu-CruPkxk6sMbXXluJK), but the whole talk can be found [here](https://www.youtube.com/watch?v=pTgIx-ucMsY) (timestamps below):
```